src/main/scala/no/ndla/searchapi/service/search/SearchConverterService.scala

```diff
@@ -737,7 +736,7 @@ trait SearchConverterService {
         relevance = relevance,
         path = context.path,
         breadcrumbs = breadcrumbs,
-        filters = filters,
+        filters = List.empty,
```

Comment:
Kan hive på en todo om å slette dette feltet dersom det skal vekk?
Burde vel kanskje versjonere endepunktet for å fjerne det men...